release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -454,6 +454,46 @@ For more information, see the following sections:
 [id="ocp-4-15-storage"]
 === Storage
 
+[id="ocp-4.15-storage-recovering-vgs-from-prev-installation"]
+==== Recovering volume groups from the previous {lvms} installation
+With this release, the `LVMCluster` custom resource (CR) provides support for recovering volume groups from the previous {lvms} installation. If the `deviceClasses.name` field is set to the name of a volume group from the previous {lvms} installation, {lvms} recreates the resources related to that volume group in the current {lvms} installation. This simplifies the process of using devices from the previous {lvms} installation through the re-installation of {lvms}.
+
+For more information, see xref:../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-creating-lvms-cluster_logical-volume-manager-storage[Creating an `LVMCluster` custom resource (CR)].
```

Comment:
This link text doesn't match the section it links to, "Creating a Logical Volume Manager cluster on a worker node". Is this text being updated in another PR? Or should this be updated here to match?

Reply:
This heading is being updated in another PR. However, that PR might not be merged in this release. It is part of the doc improvements task. 
I have updated the link text to match with the current heading.